.github/workflows/ci.yml

```diff
@@ -94,7 +96,10 @@ jobs:
   publish:
     name: Publish release artifact
     runs-on: ubuntu-latest
-    if: github.ref_type == 'tag'
+    # Conditioning based on secrets is not possible, see
+    # https://github.com/actions/runner/issues/520. Since this is a fork, we
+    # don't need to be clever about this, as we simply don't need it, ever.
+    if: 'false'
```

Comment:
Why is it `'false'`, i.e. a string, rather than `false`?

Reply:
AFAIK this needs to be an expression and therefore, a string. See https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idif.